azure/azure_style.yaml

```diff
@@ -44,17 +44,11 @@ jobs:
   pool:
     vmImage: 'ubuntu-20.04'
   continueOnError: ${{ parameters.IGNORE_STYLE }}
-  strategy:
-    matrix:
-      'python37':
-        PYTHON_VERSION: '3.7'
-      'python38':
-        PYTHON_VERSION: '3.8'
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: $(PYTHON_VERSION)
+      versionSpec: '3.8'
   - script: |
       cd ${{ parameters.REPO_NAME }}
       pip install black==20.8b1
-      python -m black . --check --diff -l 120
+      black . --check --diff -l 120 --target-version py37 --target-version py38
```

Comment:
This didn't work without `python -m` when we tried it, but I think that was before we had `UsePythonVersion@0` working properly. Maybe it's worth a shot?

Reply:
Yeah I'm like 95% sure that this works fine.

Reply:
Okay, let's try it